File: app/backend/models/comment.js
const {mongoose} =  require('../db');  // The mongodb connector library

let Comment = mongoose.model('Comment', {

  userId: {
    type: String,
    require: true
  },

  related: {
    type: String,
    require: true
  },

  text: {
    type: String,
    require: true
  },

  date:{
    type: Date,
    require: true
  },

  about:{
    type: String,
    require: true
  }
});

module.exports = {
  Comment: Comment
}

File: app/android/app/src/main/java/com/example/arken/model/Comment.kt
package com.example.arken.model

import java.io.Serializable
import java.util.*
import javax.net.ssl.X509ExtendedTrustManager

class Comment(
    val userId: String?,
    val _id: String?,
    val related: String?,
    val text: String?,
    val date: Date?,
    val about: String?,
    val username: String?,
    val usersurname: String?
) : Serializable{
    constructor(related: String, text: String, about: String) : this(null, null, related, text, null, about, null, null)
    constructor(username: String?, usersurname: String?, text: String, id: String) : this(id, null, null, text, null, null, username, usersurname)

}